genesis.json

```diff
@@ -1,5 +1,5 @@
 {
-  "genesis_time": "2021-11-22T14:53:39.956721428Z",
+  "genesis_time": "2021-11-22T14:53:13.683810227Z",
```

Comment:
I think this is wrong. I don't understand why we would need to create a genesis file separately per gentx submission.
You must use the same pregenesis file before.